### ISSUE TITLE
port issue

### ISSUE THREAD

**Issue by jefin198609** [terraform-google-modules/terraform-google-memorystore#106]
### TL;DR

when I use this module for creating memorystore, its automatically created with port 6378 instead of port 6379.

### Expected behavior

Below I mentioned the output. I want to made connection between VM and redis instance with port 6379.

auth_string = <sensitive>
create_time = ""
current_location_id = "us-central1-a"
host = "10.34.28.235"
id = ""
nodes = tolist([
  {
    "id" = "node-0"
    "zone" = "us-central1-a"
  },
])
persistence_iam_identity = ""
port = 6378

### Observed behavior

Below I mentioned the output

auth_string = <sensitive>
create_time = ""
current_location_id = "us-central1-a"
host = "10.34.28.235"
id = ""
nodes = tolist([
  {
    "id" = "node-0"
    "zone" = "us-central1-a"
  },
])
persistence_iam_identity = ""
port = 6378

### Terraform Configuration

```hcl
resource "google_redis_instance" "default" {
  project            = var.project
  name               = var.name
  tier               = var.tier
  replica_count      = var.tier == "STANDARD_HA" ? var.replica_count : null
  read_replicas_mode = var.tier == "STANDARD_HA" ? var.read_replicas_mode : null
  memory_size_gb     = var.memory_size_gb
  connect_mode       = var.connect_mode
  region                  = var.region
  location_id             = var.location_id
  alternative_location_id = var.alternative_location_id

  authorized_network   = var.authorized_network
  customer_managed_key = var.customer_managed_key

  redis_version     = var.redis_version
  redis_configs     = var.redis_configs
  display_name      = var.display_name
  reserved_ip_range = var.reserved_ip_range

  labels = var.labels

  auth_enabled = var.auth_enabled

  transit_encryption_mode = var.transit_encryption_mode

  dynamic "maintenance_policy" {
    for_each = var.maintenance_policy != null ? [var.maintenance_policy] : []
    content {
      weekly_maintenance_window {
        day = maintenance_policy.value["day"]
        start_time {
          hours   = maintenance_policy.value["start_time"]["hours"]
          minutes = maintenance_policy.value["start_time"]["minutes"]
          seconds = maintenance_policy.value["start_time"]["seconds"]
          nanos   = maintenance_policy.value["start_time"]["nanos"]
        }
      }
    }
  }
}
```

### Terraform Version

```sh
terraform {
  required_providers {
   google = {
      source = "hashicorp/google"
      version = "4.27.0"
      }
}
}
provider "google" {

}
```
